geode-gfsh/build.gradle

```diff
@@ -40,6 +40,10 @@ dependencies {
   implementation('com.fasterxml.jackson.core:jackson-databind')
   implementation('io.swagger.core.v3:swagger-annotations')
 
+  // JAXB dependencies needed for Java 11+
+  implementation('javax.xml.bind:jaxb-api')
+  implementation('com.sun.xml.bind:jaxb-impl')
```

Comment:
Should we also add activation? On Java 11+, JAXB is no longer in the JDK. This combo can work, but you often also need activation. Newer stacks prefer Jakarta (jakarta.xml.bind).

```
implementation 'javax.xml.bind:jaxb-api:2.3.1'
runtimeOnly    'com.sun.xml.bind:jaxb-impl:2.3.3'
runtimeOnly    'com.sun.activation:javax.activation:1.2.0'
```